source/fuzz/fuzzer_util.cpp

```diff
@@ -1656,9 +1673,8 @@ bool MembersHaveBuiltInDecoration(opt::IRContext* ir_context,
 bool HasBlockOrBufferBlockDecoration(opt::IRContext* ir_context, uint32_t id) {
   for (auto decoration : {SpvDecorationBlock, SpvDecorationBufferBlock}) {
     if (!ir_context->get_decoration_mgr()->WhileEachDecoration(
-            id, decoration, [](const opt::Instruction & /*unused*/) -> bool {
-              return false;
-            })) {
+            id, decoration,
+            [](const opt::Instruction& /*unused*/) -> bool { return false; })) {
```

Comment:
It looks like you changed this formatting by mistake - please revert to the original formatting. Occasionally this happens when using clang-format and I'm not quite sure why.